build/WindowsAppSDK-TDBuild.yml

```diff
@@ -21,5 +21,3 @@ jobs:
         dev\VSIX\Extension\Cpp\Common\VSPackage.resx
         dev\VSIX\Extension\Cs\Common\VSPackage.resx
       teamId: $(TdBuildTeamIDVSIX)
-      authId: $(TdBuildAuthID)
-      authKey: $(TdBuildToken)
```

Comment:
>authKey [](http://example.com/codeflow?start=6&length=7)

The parameter in the template was called `AuthToken`, not `authKey`. (Also, does the different casing matter?)

Reply:
No it was always AuthKey?